README.md

```diff
@@ -47,6 +47,7 @@ Besides [Monero](https://getmonero.org), following coins can be mined using this
 - [Intense](https://intensecoin.com)
 - [Karbo](https://karbo.io)
 - [Sumokoin](https://www.sumokoin.org)
+- [Haven](https://havenprotocol.com)
```

Comment:
This list is in alphabetical order, please insert at the proper position.

Reply:
Have updated the ordering to be alphabetical